src/plugins/discover/public/application/components/discover_legacy.tsx

```diff
@@ -203,117 +192,143 @@ export function DiscoverLegacy({
+                        />
+                      </div>
+                    )}
+                  </section>
+                )}
+
+                <div className="dscResults">
+                  <section
+                    className="dscTable dscTableFixedScroll"
+                    aria-labelledby="documentsAriaLabel"
+                    ref={scrollableDesktop}
+                  >
+                    <h2 className="euiScreenReaderOnly" id="documentsAriaLabel">
+                      <FormattedMessage
+                        id="discover.documentsAriaLabel"
+                        defaultMessage="Documents"
+                      />
+                    </h2>
+                    {rows && rows.length && (
+                      <div className="dscDiscover">
+                        <DocTableLegacy
+                          columns={state.columns || []}
+                          indexPattern={indexPattern}
+                          minimumVisibleRows={minimumVisibleRows}
+                          rows={rows}
+                          sort={state.sort || []}
+                          searchDescription={opts.savedSearch.description}
+                          searchTitle={opts.savedSearch.lastSavedTitle}
+                          onAddColumn={addColumn}
+                          onFilter={onAddFilter}
+                          onMoveColumn={onMoveColumn}
+                          onRemoveColumn={onRemoveColumn}
+                          onSort={onSort}
                         />
-                      </h2>
-                      {rows && rows.length && (
-                        <div className="dscDiscover">
-                          <DocTableLegacy
-                            columns={state.columns || []}
-                            indexPattern={indexPattern}
-                            minimumVisibleRows={minimumVisibleRows}
-                            rows={rows}
-                            sort={state.sort || []}
-                            searchDescription={opts.savedSearch.description}
-                            searchTitle={opts.savedSearch.lastSavedTitle}
-                            onAddColumn={addColumn}
-                            onFilter={onAddFilter}
-                            onMoveColumn={onMoveColumn}
-                            onRemoveColumn={onRemoveColumn}
-                            onSort={onSort}
-                          />
-                          <a tabIndex={0} id="discoverBottomMarker">
-                            &#8203;
-                          </a>
-                          {rows.length === opts.sampleSize && (
-                            <div
-                              className="dscTable__footer"
-                              data-test-subj="discoverDocTableFooter"
+                        <a tabIndex={0} id="discoverBottomMarker">
+                          &#8203;
+                        </a>
```

Comment:
This is used for the skip link on this page... And we're getting rid of all of this in the DataGrid PR but we can make this at least a little better...

- This isn't a link so we shouldn't use `a`
- We need to move focus to this but a user never needs to focus on this, so `tabIndex={-1}` is a better option than `0`

```suggestion
                        <span tabIndex={-1} id="discoverBottomMarker">
                          &#8203;
                        </span>
```

Reply:
very true & done!